addons/languages/german/pack/src/main/res/xml/de_qwerty.xml

```diff
@@ -5,15 +5,15 @@
     android:keyWidth="10%p">
     
     <Row android:keyWidth="9.09%p">
-        <Key android:codes="113" android:keyLabel="q" android:popupCharacters="\@1" android:keyEdgeFlags="left"/>
+        <Key android:codes="113" android:keyLabel="q" android:popupCharacters="1\@" android:keyEdgeFlags="left"/>
         <Key android:codes="119" android:keyLabel="w" android:popupCharacters="2"/>
-        <Key android:codes="e"   android:keyLabel="e" android:popupKeyboard="@xml/de_popup_e" ask:hintLabel="€3"/>
+        <Key android:codes="e"   android:keyLabel="e" android:popupKeyboard="3@xml/de_popup_e" ask:hintLabel="€3"/>
         <Key android:codes="114" android:keyLabel="r" android:popupCharacters="4"/>
         <Key android:codes="116" android:keyLabel="t" android:popupCharacters="5"/>
-        <Key android:codes="121" android:keyLabel="y" android:popupCharacters="ýÿ"/>
-        <Key android:codes="117" android:keyLabel="u" android:popupCharacters="ü7ùúûũūŭű"/>
+        <Key android:codes="121" android:keyLabel="y" android:popupCharacters="6ýÿ"/>
```

Comment:
Should the 6 be deleted from the popup menu oft the letter z? Or is it intended to be there additionally for this querty layout?